.taprc

```diff
@@ -0,0 +1,8 @@
+ts: false
+jsx: false
+flow: false
```

Comment:
I don't think these are needed, are they?

Reply:
No you're right! Removed them